conjur/wrapper/keystore_adapter.py

```diff
@@ -10,15 +10,24 @@
 
 # Builtins
 import logging
+import platform
+import os
 
 # Third party
 import keyring
 
-# Internals
-from conjur.constants import KEYSTORE_ATTRIBUTES
-
 # TODO should verify we are using the exact keyring version wanted and
 #  disable insecure keyring such as PlaintextKeyring.
+from conjur.errors import KeyringDeletionError, KeyringGeneralError, KeyringSetError
+
+if platform.system() == "Darwin":
```

Comment:
should this be wrapped in a function?

Please add a comment why this is outside the class and in the module. As you explained to me

Reply:
write in `util_functions.py
file:///Users/ssax/Documents/work/conjur-api-python3/conjur/util/util_functions.py`

Reply:
done